scripts/docs/generate_asset_documentation.py

```diff
@@ -133,6 +133,12 @@ def _add_doc_tags(self, doc):
             tags.sort(key=lambda x: x[0].lower())
             doc.add_raw(" ".join(tags))
         return doc
+    
+    def _add_doc_link(self, doc):
+        link = "https://ml.azure.com/registries/azureml/{}/{}/version/{}".format(
+            self.type + "s", self.name, self.version)
```

Comment:
Consider putting the 's' in the string
```suggestion
        link = "https://ml.azure.com/registries/azureml/{}s/{}/version/{}".format(
            self.type, self.name, self.version)
```